### ISSUE TITLE
Lychee not working with php-fpm

### ISSUE THREAD

**Issue by Nordikk** [LycheeOrg/Lychee#502]
I switched from php7.3 to php7.3-fpm yesterday and switched from mpm_preforked to mpm_event. That are necessary steps to use HTTP/2 with Apache.

Lychee is still displaying, but I can't upload any photos. Lychee will respond with an error "Unknown response from the server". 

I switched back to php7.3 and mpm_preforked, because I don't really need HTTP/2, but it would be nice if it would be possible in the future with Lychee. :-)

**Comment by d7415** [comment]
I'll need more information to figure out where, but it's a configuration issue. I'm using php-fpm (with nginx) without issue. Lychee working with your other configuration is good for narrowing it down, but we'll need to know more about your setup if anyone is going to solve it.

Also, I notice you deleted the issue template. Please don't do that. It's there for a reason.

**Comment by Nordikk** [comment]
What info do you need?

**Comment by d7415** [comment]
Ok, so if it's only uploading then permissions is the first thing to check - is fpm running as the same user as apache? Does it have access to the necessary folders?

Additionally, if you set `APP_DEBUG=true` in your `.env` you may be able to see more useful error messages in your browser tools' network tab.

**Comment by ildyria** [comment]
Any updates ?

**Comment by StefanescuCristian** [comment]
Hello,

I'm having the same problem. Maybe it's a coincidence but I've switched Apache to mpm_event +  php-fpm 7.4 too.
In laravel.log I have the following error when uploading a JPG file:

`[2020-05-11 18:44:49] local.ERROR: Call to undefined method PHPExif\Exif::getDescription() {"exception":"[object] (Error(code: 0): Call to undefined method PHPExif\\Exif::getDescription() at /var/www/html/photos.linux365.ro/app/Metadata/Extractor.php:113)
[stacktrace]
`

`php -m | grep exif
exif`

Is this maybe related or there's a problem in my configuration?

PS: I have PHP from ondrej PPA

**Comment by d7415** [comment]
@StefanescuCristian Was your installation working before the change? That looks like a dependency issue.

**Comment by StefanescuCristian** [comment]
I can't remember if it was working before the switch, but I've switched back to prefork and mod_php and indeed, the upload gives me the same error as with fpm.

It's strange because I have php-exif installed.

It sux that I have updated from 4.0.1 to 4.0.3 today and updated the system too, then decided to switch to FPM, and after that I wanted to upload more photos in my gallery and the web app gave me the error.

I guess my problem is not related to the original one.

EDIT: I did a `composer update` and everything works now.
